src/config.ts

```diff
@@ -15,10 +15,10 @@ export const getConfig = (request: FastifyRequest): Config => {
     : configHeader ?? "{}";
   const config = JSON.parse(rawConfigJson);
   return {
-    host: process.env[config.host] ?? config.host,
-    scheme: process.env[config.scheme] ?? config.scheme ?? "http",
-    apiKey: process.env[config.apiKey] ?? config.apiKey,
-    openApiKey: process.env[config.openApiKey] ?? config.openApiKey,
+    host: config.host,
+    scheme: config.scheme ?? "http",
+    apiKey: config.apiKey,
+    openApiKey: config.openApiKey,
```

Comment:
❓  Totally fine, but curious why these were removed?

Reply:
for better or worse, config is loaded for each query/mutation https://github.com/digs-com/weaviate_gdc/blob/c3cbf9bd37df8b37aa6292c83400639b815212f0/src/index.ts#L61 If the env var is refreshed on hasura side, that value is reflected immediately. If this is set via env var, then any changes to the env var on hasura will never be picked up. So I removed it. Can add it back if needed?